framework/caching-cmm.md

```diff
@@ -144,7 +144,7 @@ The caching CMM MUST use the formulas specified in [Appendix A](#appendix-a-cach
 in order to compute the [cache entry identifier](cryptographic-materials-cache.md#cache-identifier).
 
 If a cache entry is found, the caching CMM MUST return the encryption materials retrieved.
-If a cache entry is not found, the caching CMM MUST then attempt to obtain the encryption materials
+If a cache entry is not found or the cache entry is expired, the caching CMM MUST then attempt to obtain the encryption materials
```

Comment:
If the cache entry is found, but its hypothetically expired, this should still apply.

This matches the wording for `Decrypt Materials` on line 167 of this document.